cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -1235,4 +1235,243 @@ TYPED_TEST(GroupedTimeRangeRollingTest, SimplePartitionedStaticWindowsWithNoGrou
                          1);
 }
 
+template <typename T>
+struct TypedNullTimestampTestForRangeQueries : public cudf::test::BaseFixture {
+};
+
+struct NullTimestampTestForRangeQueries : public cudf::test::BaseFixture {
+};
+
+TYPED_TEST_CASE(TypedNullTimestampTestForRangeQueries, cudf::test::IntegralTypes);
+
+TYPED_TEST(TypedNullTimestampTestForRangeQueries, CountSingleGroupTimestampASCNullsFirst)
```

Comment:
Include tests for the case when all values are null (both single and multiple groups).

Reply:
Agreed. Added both.